src/citrine/informatics/predictors.py

```diff
@@ -66,12 +65,12 @@ def __init__(self,
                  description: str,
                  input_descriptors: List[Descriptor],
                  output_descriptor: Descriptor,
-                 session: Optional[Session] = None):
+                 session: Session = Session()):
```

Comment:
This is rarely what you want to do.  This will create a single session when the method is interpreted (not called).  Which means that all calls get the same session, and also that the default is always created even if never used.  Better to default to `None` and explicitly write how to create/use a default session.  This probably isn't a huge deal as written but as the code changes it could bite us.

Reply:
Note: this pattern is repeated in many places. We should clean that out!